app/app.js

```diff
@@ -55,7 +55,7 @@ const store = configureStore(initialState, browserHistory);
 // could be used with the devTools' store.
 // As this boilerplate uses Redux & Redux-Saga, the `updateStore` is needed
 // if you want to `take` actions in your Sagas, dispatched from devTools.
-if (window.devToolsExtension) {
+if (process.env.NODE_ENV !== 'production' && window.devToolsExtension) {
```

Comment:
@mxstbr why is this of piece of code not in the templates/app.js file? I feel this is missing should I add it as well?